### PR TITLE
System & common pkgs & fonts for all hosts

### PR DESCRIPTION
**Pull request by dwilliam62**
# Pull Request

## Description
 Currently fonts and all packages are defined per host. this file  modules/packages.nix will define a common set of packages and fonts for all defined hosts.  Each host still can have additional packages and fonts their config.nix file

## Type of change

Please put an `x` in the boxes that apply:

- [x] **New feature** (non-breaking change which adds functionality)

## Checklist

Please put an `x` in the boxes that apply:

- [X] I have read the [CONTRIBUTING](https://github.com/JaKooLit/NixOS-Hyprland/blob/main/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [X] My commit message follows the [commit guidelines](https://github.com/JaKooLit/NixOS-Hyprland/blob/main/CONTRIBUTING.md#git-commit-messages).
- [X] My change requires a change to the documentation.
- [ ] I want to add something in NixOS-Hyprland wiki.
- [ ] I have added tests to cover my changes.
- [X] I have tested my code locally and it works as expected.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by JaKooLit — CHANGES_REQUESTED** [review]
looks alright but kindly remove this
    aquamarine
    hyprlang
    hyprshot
    hyprutils
    hyprcursor  
    hyprpolkitagent
    hyprpaper
    
    seems hyprlock is also missing
    
    
    and also,
    
    kindly remove this
    
    
 ## common packages 

    alacritty
    appimage-run
    bat
    bottom
    gping
    gpu-screen-recorder
    gpu-screen-recorder-gtk
    htop
    lsd
    ncdu
    neovim
    nh
    ripgrep
    tldr
    ugrep

**Review by JaKooLit — CHANGES_REQUESTED** [review]
oh in general,

kindly align with this

https://github.com/JaKooLit/NixOS-Hyprland/pull/26/commits/71604b603b06a16acab828efef590bbff7b6cdcd

atleast for now

**Review by JaKooLit — APPROVED** [review]
thanks
